doc/lua_api.txt

```diff
@@ -3171,7 +3193,10 @@ For the following functions, `v`, `v1`, `v2` are vectors,
 * `vector.cross(v1, v2)`:
     * Returns the cross product of `v1` and `v2`.
 * `vector.offset(v, x, y, z)`:
-    * Returns the sum of the vectors `v` and `{x = x, y = y, z = z}`.
+    * Returns the sum of the vectors `v` and `(x, y, z)`.
+* `vector.is_vector(v)`:
+    * Returns `true` iff `v` is a real vector, eg. created by a `vector.*` function.
```

Comment:
`has_metatable` would be too long imo, and doesn't convey what this function is intended to be used for.
But you might be right, a different name might be better.

The "iff"s are on purpose, see https://en.wikipedia.org/wiki/If_and_only_if.

Reply:
The word "iff" might indeed be a little bit confusing as it probably seems to be a bit uncommon.

Would it be a viable option to rephrase the sentence in order to avoid this expression?

Suggestion:
```suggestion
    * Returns a boolean value indicating whether `v` is a real vector, eg. created by a `vector.*` function.
```

Reply:
Ok, I've applied you suggestion.
It would be simpler if everyone learned to use and read iff.

Reply:
I've renamed it to `is_real` now. Sounds a bit weird, but whatever.

>(BTW, I haven't forgotten about testing this -- I'm just pressed for time this week.)

(Yes, sure, no problem. : ))